README.md

```diff
@@ -52,9 +52,10 @@ The [Contributor Guide](./contributing.md) provides help on how to get your idea
 
 ## Community Release Schedule
 
-- **Release Cadence**: A new release every 3 weeks.
-- **Code Freeze**: The Monday of the release week.
-- **Release Day**: The day after code freeze.
+- **Release Cadence**: A new release every 4 weeks.
+- **Code Freeze**: Friday of the code freeze week.
+- **Release Day**: The following Monday after code freeze
+- **Relationship to RHOAI**: ODH code freeze occurs 1 week before RHOAI code freeze (also AICP Internal Code Freeze). ODH releases approximately 1 week before the RHOAI release.
```

Comment:
we should remove the section of "ODH Community Meeting - Monthly on the third Monday @ 12pm EST" 

and i do not think we should mention RHOAI in community.
just say Release cadence is about 4 weeks and Code freeze on Friday, Release the following monday (in best efforts)